### ISSUE TITLE
[Feature] Add option to use `unknown` instead of `any` in output

### ISSUE THREAD

**Issue by matthew-nichols-westtel** [bcherny/json-schema-to-typescript#275]
Aka have a schema like `{"type":"object","title":"Object"}` turn into

```
export interface Object {
  [k: string]: unknown;
}
```

instead of

```
export interface Object {
  [k: string]: any;
}
```

`unknown` is much less likely to cause issues from misspellings, etc (aka many of the problems in #153) and is just safer than `any` in general.

**Comment by dl748** [comment]
Love this feature, forces other devs to either typecast or fix the schema.  I'd prefer this as the default as its "safer" but unfortunately it prevents backwards compatibility.

**Comment by mike-marcacci** [comment]
I addressed this in #291 before noticing #281... it looks like we took slightly different approaches, but either will work.
